generator/wrappers.py

```diff
@@ -171,8 +175,8 @@ def wrap_algo(algo, ver):
     'algorithms::brownboost::training': {'weights': True,},
     'algorithms::logitboost::training': {'weights': True,},
     'algorithms::svm::training': {'weights': True,},
-    'algorithms::kdtree_knn_classification::training': {'weights': True,},
-    'algorithms::bf_knn_classification::training': {'weights': True,},
+    'algorithms::kdtree_knn_classification::training': {'weights': True, 'labels': True,},
```

Comment:
What is meant by this?

Reply:
Labels optionality support is added by these lines (same as previous `weights`)